configs/projects/puppet-agent.rb

```diff
@@ -30,6 +30,18 @@
 
   platform = proj.get_platform
 
+  # HuaweiOS is a cross-compiled platform
+  if platform.is_huaweios?
+    platform_triple = "powerpc-unknown-linux-gnu"
+    host = "--host #{platform_triple}"
+
+    # Use a standalone ruby for cross-compilation
+    proj.setting(:host_ruby, "/opt/pl-build-tools/bin/ruby")
+    proj.setting(:host_gem, "/opt/pl-build-tools/bin/gem")
+
+    proj.noarch
```

Comment:
I'd like to add proper support for building PPC deb packages, but there is a lot of invasive work to do in vanagon for that. Also, setting noarch on the package avoids stripping the binaries, which is what we want for initial testing. So I'd say these are reasonable compromises for getting this into the repo in a timely manner.